robotd/devices.py

```diff
@@ -97,6 +97,30 @@ def status(self):
         return {'temperature': temp_milli_degrees / 1000}
 
 
+class GameState(Board):
```

Comment:
I'm not sure about the semantic overload of `Board` here. Could we do something else or rename `Board` and clarify what it covers?